front/src/components/BackOfficeNavPM.jsx

```diff
@@ -0,0 +1,36 @@
+import routes from "@/routes"
+import { ClockIcon, RectangleGroupIcon, ShoppingCartIcon, TagIcon } from "@heroicons/react/24/outline"
+import clsx from "clsx"
+import Link from "next/link"
+
+const BackOfficeNavPM = ({ page }) => (
```

Comment:
You repeat yourself
Each element of the Nav is a li that has a Link with an icon and a text.
Can you make a component ?